files/similar/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "similar",
+  "repo": "InputSimilar/similar",
```

Comment:
The tags here are not numbers https://github.com/InputSimilar/similar/releases
It wont work